swagger.yml

```diff
@@ -355,6 +355,11 @@ paths:
           description: The id of the order for which tracking information is being requested.
           required: true
           type: string
+        - name: showPackageTracking
+          in: query
+          description: Flag for returning the package-level tracking info in response
+          required: true
```

Comment:
optional, default false

Reply:
Also, Tracking Response.timeline now includes a new field `trackingNumber` which needs to be represented in the docs